src/windows/views/transitions_treeview.py

```diff
@@ -58,7 +58,6 @@ def startDrag(self, event):
             current = selected[0]
 
         if not current.isValid():
-            # We can't find anything to drag
```

Comment:
:rofl: Thank you for helping delivering us, at least in part, from the tedium of narrative comments. These make my teeth grind. There was one I saw recently, it was something very close to this:
```python
# Emit SomeSignal
self.SomeSignal.emit()
```
Which just makes me want to scream.

Reply:
(Naturally, having said this, a quick `git blame` informs me the commit that added the "We can't find anything to drag" comments came from... hmm... some guy named @ferdnyc. Typical. He's the worst.)